src/Lib/Modes/PathEditMode.vala

```diff
@@ -312,7 +312,6 @@ public class Akira.Lib.Modes.PathEditMode : AbstractInteractionMode {
 
         bool is_selected = edit_model.hit_test (event.x, event.y, ref sel_point, ref underlying_pt);
 
-        bool is_shift = (event.state == Gdk.ModifierType.SHIFT_MASK);
         bool is_alt = (event.state == Gdk.ModifierType.MOD1_MASK);
```

Comment:
@AshishS-1123 I can't figure out what do these shift and alt modifiers do while in editing mode.
Could you give me an example?